examples/basic_room/index.ts

```diff
@@ -1,6 +1,44 @@
-import { Room } from '@livekit/rtc-node';
+import { readFileSync } from 'node:fs';
+import {
+  AudioFrame,
+  AudioSource,
+  LocalAudioTrack,
+  Room,
+  TrackPublishOptions,
+  TrackSource,
+  dispose,
+} from '@livekit/rtc-node';
+import { AccessToken } from 'livekit-server-sdk';
+import { join } from 'node:path';
 
+// create access token from API credentials
+const token = new AccessToken(process.env.LIVEKIT_API_KEY, process.env.LIVEKIT_API_SECRET, {
+  identity: 'example-participant',
+})
+token.addGrant({
+  room: 'example-room',
+  roomJoin: true,
+  roomCreate: true,
+});
+const jwt = await token.toJwt();
+
+// set up room
 const room = new Room();
-room.connect('ws://localhost:7880', 'my-token');
+await room.connect(process.env.LIVEKIT_URL, jwt, { autoSubscribe: true, dynacast: true });
+console.log('connected to room', room);
+
+// set up audio track
+const source = new AudioSource(16000, 1);
+const track = LocalAudioTrack.createAudioTrack('audio', source);
+const options = new TrackPublishOptions();
+options.source = TrackSource.SOURCE_MICROPHONE;
+
+// read file into Uint16Array
+const sample = readFileSync(join(import.meta.dirname, '../speex.wav'));
```

Comment:
You can add a basic wav reader on the example, it doesn't have to be fancy, just to correctly read the used sample_rate and the number of channels.